src/configure.ac

```diff
@@ -976,10 +976,24 @@ AC_ARG_ENABLE(dev,
 
 ##############################################################################
 # Subsection 2.3.1                                                           #
-# protobuf, zmq, websockets, json, uriparse, avahi-client packages           #
+# CK, protobuf, zmq, websockets, json, uriparse, avahi-client packages           #
 # NML
 ##############################################################################
 
+PKG_CHECK_MODULES([CK], [ck >= 0.4.6],
+    [
+	AC_DEFINE(HAVE_CK, [], [concurrencykit found])
+	# needed for kbuild:
+        CK_INCLUDE=`pkg-config --cflags-only-I ck`
+        USE_CK=yes
+	AC_MSG_RESULT([concurrencykit found - configuring to use ck])
+    ],
+    [
+        USE_CK=no
+	AC_MSG_ERROR([ concurrencykit not found - please install package libck0 ])
```

Comment:
This will cause the build to exit if `libck` isn't available.  Since `libck` isn't shipped for Jessie i386, can this line just be dropped, and `libck` be optional?

Reply:
Ah ha, turns out the version of `libck` in Jessie is 0.3.5, whereas this patch requires `PKG_CHECK_MODULES([CK], [ck >= 0.4.6], [...])`.  Backing this down to 0.3.5 and building in Jessie prints lots of errors like this:

```
hal/lib/hal_accessor.h:258:230: error: ‘ck_pr_load_double’ was not declared in this scope
```

So I think `libck` must be optional at least as long as we support Jessie.